std/traits.d

```diff
@@ -624,13 +625,15 @@ private template fqnType(T,
         static if (attrs == FA.none)
             return "";
         else
-            return format("%s%s%s%s%s%s",
+            return format("%s%s%s%s%s%s%s%s",
                  attrs & FA.pure_ ? " pure" : "",
                  attrs & FA.nothrow_ ? " nothrow" : "",
                  attrs & FA.ref_ ? " ref" : "",
                  attrs & FA.property ? " @property" : "",
                  attrs & FA.trusted ? " @trusted" : "",
-                 attrs & FA.safe ? " @safe" : ""
+                 attrs & FA.safe ? " @safe" : "",
+                 attrs & FA.nogc ? " @nogc" : "",
+                 attrs & FA.return_ ? " return" : ""
```

Comment:
Let's remember that D allows trailing commas, specifically for when you have a "list style" layout such as here. This is just a comment.